packages/block-library/src/gallery/gallery-image.js

```diff
@@ -102,7 +102,7 @@ class GalleryImage extends Component {
 		// Disable reason: Image itself is not meant to be
 		// interactive, but should direct image selection and unfocus caption fields
 		// eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/click-events-have-key-events
-		const img = url ? <img src={ url } alt={ alt } data-id={ id } onClick={ this.onImageClick } /> : <Spinner />;
+		const img = url ? <img src={ url } alt={ alt } data-id={ id } onClick={ this.onImageClick } tabIndex="0" onKeyDown={ this.onImageClick } /> : <Spinner />;
```

Comment:
Does this allow us to drop some of the disabled ESLint rules on the preceding line, e.g. `jsx-a11y/click-events-have-key-events` ?

Reply:
Good catch, but as it turns out no. I'm getting this error when I remove it:

```
{
	"resource": "/Users/joen/GIT/gutenberg/packages/block-library/src/gallery/gallery-image.js",
	"owner": "eslint",
	"code": "jsx-a11y/no-noninteractive-element-interactions",
	"severity": 8,
	"message": "Non-interactive elements should not be assigned mouse or keyboard event listeners.",
	"source": "eslint",
	"startLineNumber": 102,
	"startColumn": 21,
	"endLineNumber": 102,
	"endColumn": 142
}
```

![screenshot 2018-10-30 at 13 56 03](https://user-images.githubusercontent.com/1204802/47719489-a1cc9f80-dc4b-11e8-9291-afd42ece6bb5.png)

Reply:
Is that with the whole line removed? What about just the revision of https://github.com/WordPress/gutenberg/pull/11205/files#r229301935 ?

Reply:
As usual, you are right. Your suggestion passes. Thank you.